book/02-git-basics/sections/viewing-history.asc

```diff
@@ -227,7 +228,7 @@ Those are only some simple output-formatting options to `git log` – there are
 ==== Limiting Log Output
 
 In addition to output-formatting options, `git log` takes a number of useful limiting options – that is, options that let you show only a subset of commits.
-You've seen one such option already – the `-2` option, which show only the last two commits.
+You've seen one such option already, the `-2` option, which shows only the last two commits.
```

Comment:
This seems like a good place for a colon though, right? A comma doesn't quite feel right to me.